test-tools/issuer-front-end/src/Main.tsx

```diff
@@ -1661,14 +1674,263 @@ export default function Main(props: WalletConnectionProps) {
                                     </div>
                                 )}
                             </TestBox>
+                            <TestBox
+                                header="Step 13: Register a credential (with wrong attributes)"
+                                note="Expected result after pressing the button: There should be one popup happening in the wallet
+                                    (first action to add the credential, second action to send the `issueCredential` tx should not appear because the flow has already thrown an error).
+                                    The browser wallet should not allow you to add such a credential.
+                                    Pressing the button without any user input will create an example tx with the provided placeholder values.
+                                    Your credential should NOT be shown in the `Verifiable Credential` 
```

Comment:
I found the use of `tx`/transaction here confusing, since the credential issuing in the first step is not a transaction.